armi/utils/__init__.py

```diff
@@ -38,37 +37,33 @@
 
 def getFileSHA1Hash(filePath, digits=40):
```

Comment:
@john-science this was deceptively easy to do but I want your sanity check on this. Here's my understanding. The single file approach behaves the same as before. The dir approach does this:

1. loop through each file in dir
2. update sha1

The unit test does this with 2 files. To demonstrate how this was working I printed things out. Here's a simplification of that:
```
File1 sha: aabbcc
File2 sha: xxyyzz
Dir sha with update: axbycz
```

Is this acceptable behavior? No one but a downstream plugin will be using this dir option for now.

Reply:
The approach makes sense to me.  Let me keep looking.